lib/Inst/Inst.cpp

```diff
@@ -654,6 +654,43 @@ Inst *InstContext::getInst(Inst::Kind K, unsigned Width,
   return N;
 }
 
+Inst *InstContext::getInst(Inst::Kind K, unsigned Width,
```

Comment:
are you duplicating the whole body of this function? don't do that, just make one version call the other one